appinfo/Migrations/Version20170329194544.php

```diff
@@ -0,0 +1,54 @@
+<?php
+namespace OCA\oauth2\Migrations;
+
+use Doctrine\DBAL\Exception\UniqueConstraintViolationException;
+use OCA\OAuth2\Db\Client;
+use OCA\OAuth2\Db\ClientMapper;
+use OCP\Migration\ISimpleMigration;
+use OCP\Migration\IOutput;
+
+/**
+ * This adds the default client ids for ownCloud mobile and desktop clients
+ */
+class Version20170329194544 implements ISimpleMigration {
+
+	private static $registry = [
+		['Desktop Client', 'http://localhost:*', 'xdXOt13JKxym1B1QcEncf2XDkLAexMBFwiT9j6EfhhHFJhs2KM9jbjTmf8JBXE69', 'UBntmLjC2yYCeHwsyj73Uwo9TAaecAetRwMw0xYcvNL9yRdLSUi0hUAHfvCHFeFh'],
+		['Android', 'oc://android.owncloud.com', 'e4rAsNUSIUs0lF4nbv9FmCeUkTlV9GdgTLDH1b5uie7syb90SzEVrbN7HIpmWJeD', 'dInFYGV33xKzhbRmpqQltYNdfLdJIfJ9L5ISoKhNoT9qZftpdWSP71VrpGR9pmoD'],
+//		['iOS', '???', 'mxd5OQDk6es5LzOzRvidJNfXLUZS2oN3oUFeXPP8LpPrhx3UroJFduGEYIBOxkY1', 'KFeFWWEZO9TkisIQzR3fo7hfiMXlOpaqP8CFuTbSHzV1TUuGECglPxpiVKJfOXIx']
```

Comment:
Could we use oc://ios.owncloud.com as callback URL for the iOS client? The old iOS code is not testable right now, but the URI should work anyway.